s3backgrounddelete/s3backgrounddelete/s3backgroundproducer

```diff
@@ -9,5 +9,15 @@ if (len(sys.argv)>1):
 else:
   producer_name = "prod1"
 
-scheduler = ObjectRecoveryScheduler(producer_name)
+if (len(sys.argv)>2):
```

Comment:
We should use the python command line parser instead of replying on argument sequence